### ISSUE TITLE
Added gutters do not handle refresh gracefully

### ISSUE THREAD

**Issue by rundis** [LightTable/LightTable#2246]
gutter width and offset of editor content gets mangled when refresh triggers on editor.

Steps to reproduce
1. Install lt-js-hint plugin 
2. Open a new blank editor -> set current editor syntax to javascript -> write some crappy js so you get lint markers in the gutter
3. Open user behaviors and save it 
4. Voila your gutter should look pretty messed up

This whole logic seems flawed/fragile to me:
https://github.com/LightTable/LightTable/blob/master/src/lt/objs/editor.cljs#L445

**Comment by bbbates** [comment]
Yes, I noticed this in the course of playing around with the gutter marks for those linting plugins. 
If it helps the debugging effort: if you switch editors (next-tab, then previous-tab commands) it seems to refresh the gutter state, and everything looks like it should.
